pillar/heroku/mitopen.sls

```diff
@@ -93,22 +92,22 @@ heroku:
     AWS_SECRET_ACCESS_KEY: __vault__:cache:aws/creds/mitopen>data>secret_key
     AWS_STORAGE_BUCKET_NAME: 'ol-mitopen-app-storage-{{ env_data.env_name }}'
     CELERY_WORKER_MAX_MEMORY_PER_CHILD: {{ env_data.CELERY_WORKER_MAX_MEMORY_PER_CHILD }}
-    CKEDITOR_ENVIRONMENT_ID:  __vault__::secret-{{ business_unit }}/ckeditor>data>environment_id
-    CKEDITOR_SECRET_KEY:  __vault__::secret-{{ business_unit }}/ckeditor>data>secret_key
-    CKEDITOR_UPLOAD_URL:  __vault__::secret-{{ business_unit }}/ckeditor>data>upload_url
+    CKEDITOR_ENVIRONMENT_ID:  __vault__::secret-mitopen/secrets>data>data>ckeditor>environment_id
+    CKEDITOR_SECRET_KEY:  __vault__::secret-mitopen/secrets>data>data>ckeditor>secret_key
+    CKEDITOR_UPLOAD_URL:  __vault__::secret-mitopen/secrets>data>data>ckeditor>upload_url
     CSAIL_BASE_URL: https://cap.csail.mit.edu/
     DATABASE_URL: postgres://{{ pg_creds.data.username }}:{{ pg_creds.data.password }}@{{ rds_endpoint }}/mitopen
     DEBUG: {{ env_data.DEBUG }}
     DUPLICATE_COURSES_URL: https://raw.githubusercontent.com/mitodl/open-resource-blacklists/master/duplicate_courses.yml
     EDX_API_ACCESS_TOKEN_URL: https://api.edx.org/oauth2/v1/access_token
-    EDX_API_CLIENT_ID: __vault__::secret-{{ business_unit }}/edx-api-client>data>id
-    EDX_API_CLIENT_SECRET: __vault__::secret-{{ business_unit }}/edx-api-client>data>secret
+    EDX_API_CLIENT_ID: __vault__::secret-mitopen/secrets>data>data>edx-api-client>id
+    EDX_API_CLIENT_SECRET: __vault__::secret-mitopen/secrets>data>data>edx-api-client>secret
     EDX_API_URL: https://api.edx.org/catalog/v1/catalogs/10/courses
-    EMBEDLY_KEY: __vault__::secret-operations/{{ business_unit }}/embedly_key>data>value
+    EMBEDLY_KEY: __vault__::secret-operations/mitopen/embedly_key>data>value
     ENABLE_INFINITE_CORRIDOR: {{ env_data.ENABLE_INFINITE_CORRIDOR }}
     GA_G_TRACKING_ID: {{ env_data.GA_G_TRACKING_ID }}
     GA_TRACKING_ID: {{ env_data.GA_TRACKING_ID }}
-    GITHUB_ACCESS_TOKEN: __vault__::secret-{{ business_unit }}/global/odlbot-github-access-token>data>value
+    GITHUB_ACCESS_TOKEN: __vault__::secret-mitopen/secrets>data>data>global>odlbot-github-access-token
```

Comment:
We should probably just use `secret-operations` for any global values like this

Reply:
So, I'm happy to do that, but..... nothing under `secret-operations` is currently managed. That is, everything there appears to have been populated by hand. 

I can make us a managed space (probably a new mount rather than trying to pull `secret-operations` into management) for global+shared secrets and then begin populating those global+shared secrets with a `substructure` stack or something. 

I don't think it is a good idea to create a global+shared mount and then populate it with values willy-nilly from all kinds of various `application` stacks like this one, if that makes sense.

Reply:
I think that is a very sensible argument and approach. We have a general need of migrating our existing static secret mounts into more holistic management, as well as moving off of `generic` and `kv1` mounts where possible, but that's a bigger effort than necessary for this specific need.

Reply:
Okay I will populate these into `secret-operations` manually for now and open an issue to setup a `substructure` stack and begin populating it.